tests/unit/getTestTypesFunction.unitTest.ts

```diff
@@ -19,6 +21,49 @@ describe("getTestTypes function", () => {
     });
   });
 
+  context(
+    "gets successful response with suggested test types from Service",
+    () => {
+      it("returns 200 OK + data", async () => {
+        const mockTestTypesRecord = cloneDeep(TestTypes[0]);
+
+        TestTypesService.prototype.getTestTypesList = jest
+          .fn()
+          .mockResolvedValue(mockTestTypesRecord);
+        const output = await getTestTypes({}, ctx, () => {
+          return;
+        });
+        expect(output.statusCode).toEqual(200);
+        expect(JSON.parse(output.body).id).toEqual("1");
+        expect(JSON.parse(output.body).linkedIds).toEqual(["38", "39"]);
+        expect(JSON.parse(output.body).suggestedTestTypeIds).toEqual([
+          "1",
+          "7",
+          "10",
+        ]);
+        expect(JSON.parse(output.body).name).toEqual("Annual test");
```

Comment:
Looks good 👍
We could further simplify this and avoid parsing each key of the response
```js
const { name, suggestedTestTypeIds, ... } = JSON.parse(output.body)
...
expect(name).toEqual("Annual test")
...
```

 or instead check the key of the response instead of its value as this what will be used in the FE app - typescript already offers static type checking when defining interfaces.
It is worth having a look at this: https://jestjs.io/docs/expect#tohavepropertykeypath-value
Will leave it to you